.github/workflows/tests.yml

```diff
@@ -755,13 +745,13 @@ jobs:
           Path(environ["GITHUB_ENV"]).write_text(f"ANACONDA_ORG_LABEL={label}")
 
       - name: Create & Upload
-        uses: conda/actions/canary-release@v24.2.0
+        uses: conda/actions/canary-release@v24.5.0
```

Comment:
```suggestion
        uses: conda/actions/canary-release@976289d0cfd85139701b26ddd133abdd025a7b5f # v24.5.0
```

Reply:
I'll do this in a subsequent PR so we pin everything and add dependabot, thanks!